src/man/sssd-simple.5.xml

```diff
@@ -88,18 +99,36 @@
                     <term>simple_deny_users (string)</term>
                     <listitem>
                         <para>
-                            Comma separated list of users who are explicitly
-                            denied access.
+                            Comma-separated list of users who are explicitly
+                            denied access. If this option is specified, these
+                            users will be denied regardless of whether they
+                            appear in `simple_allow_users` or `simple_allow_groups`.
+                        </para>
+                        <para>
+                            **OR Logic Applies:** A user will be denied access
+                            if they are listed in `simple_deny_users` **or**
+                            if they are a member of a group in `simple_deny_groups`.
                         </para>
                     </listitem>
                 </varlistentry>
+
                 <varlistentry>
                     <term>simple_allow_groups (string)</term>
                     <listitem>
                         <para>
-                            Comma separated list of groups that are allowed to
-                            log in. This applies only to groups within this
-                            SSSD domain. Local groups are not evaluated.
+                            Comma-separated list of groups that are allowed to
+                            log in. If this option is specified, all other
+                            users are denied unless they are explicitly listed
+                            in `simple_allow_users`.
+                        </para>
+                        <para>
+                            **OR Logic Applies:** A user can log in if they
+                            are listed in `simple_allow_users` **or** if they
+                            belong to a group in `simple_allow_groups`.
+                        </para>
+                        <para>
+                            This applies only to groups within this SSSD domain.
+                            Local groups are not evaluated.
```

Comment:
IMO this part is implied, as we define `simple_*` options in the SSSD domain section of sssd.conf

Reply:
I don't know how frequent people face this issue, thus I added it. Maybe a second opinion?

Reply:
Sure, second opinion is okay with me.

Reply:
@sumit-bose @alexey-tikhonov any opinions?

Reply:
Hi,

I think it is good to mention this explicitly. However, I would like to suggest to not "bury" this information in the description of an option but mention this in the "DESCRIPTION" section right after "..... grants or denies access based on an access or deny list of user or group names."

Additionally, "Local groups are not evaluated." might not be true if the proxy-provider with libnss_files is used. It might be better to say something like "Groups from other domains configured in sssd.conf, even if the simple access provider is used there as well, and groups managed outside of SSSD are not evaluated."

HTH

bye,
Sumit

Reply:
Hopefully I was able to convey the message. Can you please take a look?

Reply:
Yes, thank you, looks good, ACK.